async_upnp_client/search.py

```diff
@@ -38,12 +38,16 @@ def __init__(
         target_ip: Optional[IPvXAddress] = None,
         timeout: int = SSDP_MX,
         service_type: str = SSDP_ST_ALL,
+        target: Optional[AddressTupleVXType] = None,
```

Comment:
This seems duplicate with `target_ip`. Can we unify these?

Reply:
I changed `target_ip` to `target` to allow accepting a port as well and updated the cli